src/domain/platform/admin/templates/CommunityGuidelines/CommunityGuidelinesTemplateForm.tsx

```diff
@@ -58,8 +61,10 @@ const CommunityGuidelinesTemplateForm = ({
   visual,
   onSubmit,
   actions,
+  guidelinesTemplateId,
 }: CommunityGuidelinesTemplateFormProps) => {
   const { t } = useTranslation();
+  const { spaceId } = useSpace();
```

Comment:
to fallback to the space StorageBucket. (store the files in the template's SB if we are editing, or in the space's SB if we are creating a new template)